Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -17,6 +19,8 @@ namespace Ploeh.AutoFixture
     /// </remarks>
     public class FreezeOnMatchCustomization : ICustomization
     {
+        private object request;
```

Comment:
That is not one but two new constructors have to be defined to replace the [existing](https://github.com/sergeyshushlyapin/AutoFixture/blob/e3d79a8b583b3fe7783471b0b9897ce6ac2b641b/Src/AutoFixture/FreezeOnMatchCustomization.cs#L36) [two](https://github.com/sergeyshushlyapin/AutoFixture/blob/e3d79a8b583b3fe7783471b0b9897ce6ac2b641b/Src/AutoFixture/FreezeOnMatchCustomization.cs#L52).

Making the `Request` property protected and further configuring it in a derived _internal_ customization class in a Glue Library will solve the mutation issues (and will prevent the public API from changes).